lua/lazyvim/plugins/extras/util/chezmoi.lua

```diff
@@ -0,0 +1,79 @@
+vim.notify("Hello")
+return {
+  {
+    -- highlighting for chezmoi files template files
+    "alker0/chezmoi.vim",
+    opts = {
+      use_tmp_buffer = true,
+      source_dir_path = os.getenv("HOME") .. "/.local/share/chezmoi",
+    },
+    config = function(_, opts)
+      for key, value in pairs(opts) do
+        vim.g["chezmoi#" .. key] = value
+      end
+    end,
```

Comment:
```suggestion
	init = function()
		vim.g.use_tmp_buffer = 1
		vim.g.source_dir_path = os.getenv("HOME") .. "/.local/share/chezmoi",
	end,
```
This might be better?